gamemodes/terrortown/gamemode/client/cl_vskin/vgui/dform_ttt2.lua

```diff
@@ -416,7 +417,7 @@ function PANEL:MakeSlider(data)
         left:DockMargin(left:GetIndentationMargin(), 0, 0, 0)
     end
 
-    return left
+    return right, left
```

Comment:
Note: while this looks like a breaking change, I don't think anybody used the return value of that function so far